fastapi_server/main.py

```
@@ -54,6 +56,152 @@
+    """
+    url = f"{LLAMA_BASE.rstrip('/')}/api/v1/parsing/upload"
+    data: Dict[str, str] = {}
+    if LLAMA_ORG_ID:
+        data["organization_id"] = LLAMA_ORG_ID
+    if LLAMA_PROJECT_ID:
+        data["project_id"] = LLAMA_PROJECT_ID
+
+    files = {"file": (file_path.name, file_path.open("rb"), mime)}
+    r = await client.post(url, headers=_auth_headers(), data=data, files=files, timeout=60)
+    if r.status_code >= 400:
+        raise LlamaCloudError(f"LlamaParse failed: {r.status_code} {r.text}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)